specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -451,10 +704,33 @@
             "$ref": "#/definitions/PrivateLinkServiceConnection"
           },
           "description": "A grouping of information about the connection to the remote resource. Used when the network admin does not have access to approve connections to the remote resource."
+        },
+        "customDnsConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomDnsConfig"
```

Comment:
CustomDnsConfigPropertiesFormat

Reply:
done, will push once I'm done with rest of the comments.